Source/WebCore/Modules/async-clipboard/ClipboardItem.cpp

```diff
@@ -92,6 +98,20 @@ void ClipboardItem::getType(const String& type, Ref<DeferredPromise>&& promise)
     m_dataSource->getType(type, WTFMove(promise));
 }
 
+bool ClipboardItem::supports(const String& type)
+{
+    // FIXME: Custom format starts with `"web "`("web" followed by U+0020 SPACE) prefix
+    // and suffix (after stripping out `"web "`) passes the parsing a MIME type check.
+    if (type == textPlainContentTypeAtom()
+        || type == textHTMLContentTypeAtom()
+        || type == "image/png"_s
+        || type == "text/uri-list"_s
+        || type == "image/svg+xml"_s) {
```

Comment:
Is this right? (since we don't support sanitized copy/paste of SVG data)

Reply:
Hmm indeed. I need to remove and create a FIXME in there, probably with a bug on webkit.org.

Reply:
Opened https://bugs.webkit.org/show_bug.cgi?id=280726